cypress/support/commands/commands.ts

```diff
@@ -27,6 +27,15 @@ Cypress.Commands.add('keyboardControls', (triggerKeys: any = {}, count = 1) => {
   }
 });
 
+/**
+ * wrapper around cypress.contains to search for exact strings eg find ONLY spans containing 'test-resource' and not 'test-resource-123-abc'
+ */
+Cypress.Commands.add('containsExactly', (searchString) => {
```

Comment:
We have PO methods that should probably be updated to use this, eg https://github.com/rancher/dashboard/blob/master/cypress/e2e/po/components/sortable-table.po.ts#L108-L114 but I could see that being its own PR